app/src/main/java/com/tapglue/tapgluetests/TestActivity.java

```diff
@@ -627,6 +781,83 @@ public void run() {
                 mTestController.log("Tests finished correctly", false, false);
             }
         });
+
+//        alternate search test #1
```

Comment:
Please don't commit this or enable this as a test. There's no reason to have so much dead code.